torchvision/transforms/functional_tensor.py

```diff
@@ -15,6 +15,10 @@ def _assert_image_tensor(img: Tensor) -> None:
     if not _is_tensor_a_torch_image(img):
         raise TypeError("Tensor is not a torch image.")
 
+def _assert_threshold(img: Tensor, threshold:int) -> None:
+    bound = 1 if img.is_floating_point() else 255
+    if threshold >= bound:
```

Comment:
Are you sure we need equality here? Is the upper bound a permitted value?

Reply:
I add equality because threshold should be less than upper bound.
It was bit confusing to me when I though what happens if threshold is 255 which is same as upper bound (255).

Reply:
I think the upper bound should be a permitted value. See:
https://github.com/pytorch/vision/blob/40c3ba24bfea6de02e6e1b0bd10b09d978e70506/torchvision/transforms/autoaugment.py#L226

Reply:
Thanks! Let me check and resend the PR :)